pkg/mount/block/blockdev.go

```diff
@@ -474,7 +475,12 @@ func (b BlockDevices) FilterPartType(guid string) BlockDevices {
 				continue
 			}
 			if strings.ToLower(part.Type.String()) == strings.ToLower(guid) {
-				names = append(names, fmt.Sprintf("%s%d", device.Name, i+1))
+				r := []rune(device.Name[len(device.Name)-1:])
+				if unicode.IsDigit(r[0]) {
```

Comment:
oof, i can see how what I did was a hack... can you do it for `FilterPartID` too?

Reply:
Sure.